pkgs/top-level/all-packages.nix

```diff
@@ -30511,7 +30511,7 @@ with pkgs;
   weston = callPackage ../applications/window-managers/weston { };
 
   whalebird = callPackage ../applications/misc/whalebird {
-    electron = electron_14;
+    electron = electron_18;
```

Comment:
Forgot about this one, as Electron 18 is the default, you can remove this part, or at least it would be redundant

Reply:
Done.